src/actions/AuthActions.js

```diff
@@ -27,18 +43,30 @@ const attemptFacebookLogin = async ({dispatch, navigate}) => {
     if (type === 'success') {
         let {data: {email, name, picture: {data: {url}}}} = await axios.get(`https://graph.facebook.com/me?fields=id,name,email,picture&access_token=${token}`);
 
-        dispatch({
-            type: FACEBOOK_LOGIN_SUCCESS,
-            payload: {token, appReady: false}
-        });
```

Comment:
This block below is what's new. I don't love using async/await above and then promises below, but I couldn't immediately think about how to make this cleaner with the specific flow it has. I'm open to refactoring it later, if we think of something better.

Reply:
The flow below should be pretty self explanatory, but the basic idea is
 * Try to sign the user in
 * If that works, then we're good to go
 * If it fails, check if it's because we don't have an account for them
 * If we don't have an account, try to create one